prebuilt-tasks/pom.xml

```diff
@@ -30,6 +30,10 @@
     </repositories>
 
     <dependencies>
+        <dependency>
```

Comment:
pls move next to other `org.springframework.boot` group dependencies

Reply:
fixed